lib/ruby_indexer/lib/ruby_indexer/visitor.rb

```diff
@@ -105,7 +122,12 @@ def collect_comments(node)
         comment = @comments_by_line[line]
         break unless comment
 
-        comments.unshift(comment.location.slice)
+        comment_content = comment.location.slice
+        next if comment_content.match?(IGNORED_MAGIC_COMMENTS)
+
+        comment_content.delete_prefix!("#")
+        comment_content.delete_prefix!(" ")
```

Comment:
nit: the regexp matches on `\s*` after the `#` but we're only removing one space?

Reply:
Removing more space may mess up with Markdown formatting:

```rb
# Here's my method documentation.
# 
# With an example:
# 
#     add(1, 2) # => 3
def add(val1, val2); end
```